charts/cloudzero-agent/templates/deploy.yaml

```diff
@@ -30,6 +30,36 @@ spec:
       priorityClassName: "{{ .Values.server.priorityClassName }}"
 {{- end }}
       serviceAccountName: {{ template "cloudzero-agent.serviceAccountName.server" . }}
+      initContainers:
+        - name: env-validator
+          image: python:3.12-slim
+          workingDir: /app
+          # TODO: create container image, root required to install packages atm
+          securityContext:
+            runAsUser: 0
```

Comment:
I think some users may have objections with this securitycontext. is there an easy way to work around that in the pip install? maybe installing with `--user` or similar?

Reply:
I actually didn't want to do this - but until I have the container (dockerfile and image) I needed to be root for now to install the requirements. Even as nonroot I couldn't create the venv 🫤